### PR TITLE
Improve Tasks menu

### PR DESCRIPTION
**Pull request by aminelouati**

### CODE REVIEW

File: app/templates/java.xml [hunk 1]
```diff
@@ -2,12 +2,20 @@
 <job xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="urn:proactive:jobdescriptor:3.12"
 	xsi:schemaLocation="urn:proactive:jobdescriptor:3.12 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.12/schedulerjob.xsd"
 	name="Java_Job">
+	<description>
+		A workflow that executes Java classes.
```
Comment (mboussaa) [inline app/templates/java.xml]: Add <![CDATA[ ... ]]> to the description tag. It is missing, right?

File: app/templates/java.xml [hunk 2]
```diff
@@ -2,12 +2,20 @@
...
+	</description>
 	<genericInformation>
-		<info name="var1" value="val1" />
-		<info name="var2" value="val2" />
+		<info name="Documentation" value="user/ProActiveUserGuide.html#_java_tasks"/>
 	</genericInformation>
 	<taskFlow>
 		<task name="Java_Task" maxNumberOfExecution="4">
+			<description>
+				A task that executes Java classes.
```
Comment (mboussaa) [inline app/templates/java.xml]: same

File: app/templates/script_cpython.xml [hunk 3]
```diff
@@ -3,6 +3,9 @@
 	xsi:schemaLocation="urn:proactive:jobdescriptor:3.12 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.12/schedulerjob.xsd"
 	name="Script_CPython" priority="normal" onTaskError="continueJobExecution"
 	maxNumberOfExecution="2">
+	<description>
+		A workflow that executes Python in JVM. It requires an installation of CPython on the target machine as well as the py4j module.
```
Comment (mboussaa) [inline app/templates/script_cpython.xml]: same and `Py4J`

File: app/templates/script_php.xml [hunk 4]
```diff
@@ -3,10 +3,13 @@
...
 	name="script_php" priority="normal" onTaskError="continueJobExecution"
 	maxNumberOfExecution="2">
+	<description>
+		<![CDATA[ A workflow that executes a php script.]]>
+	</description>
 	<taskFlow>
 		<task name="PHP_Task">
 			<description>
-                <![CDATA[ Executes a php script.]]>
+                <![CDATA[ A task that executes a php script.]]>
```
Comment (mboussaa) [inline app/templates/script_php.xml]: PHP

File: app/templates/java.xml (hunk 1 above)
Comment (aminelouati) [reply app/templates/java.xml]: It is there.

Comment (mboussaa) [reply app/templates/java.xml]: I mean CDATA

**Pull request merged**